landing-page/content/common/blogs.md

```diff
@@ -24,10 +24,16 @@ disableSidebar: true
 
 Here is a list of company blogs that talk about Iceberg. The blogs are ordered from most recent to oldest.
 
+
+### [From Hive Tables to Iceberg Tables: Hassle-Free](https://blog.cloudera.com/from-hive-tables-to-iceberg-tables-hassle-free/)
+**Date**: July 14rd, 2023, **Company**: Cloudera
```

Comment:
```suggestion
**Date**: July 14th, 2023, **Company**: Cloudera
```
could you also please update the `July 13rd` to `July 13th` entry in L34?

Reply:
Fixed. Thank you for spotting this.